oaknational/Oak-Web-Application
# [pupil-675] & [pupil-674] New pupil browse nav updates

## Description

Nav changes to make new pupil experience live

**Burger menu** 
 - link to /pupils/years
 - text changed to "home" 
 - no external icon
 
**Footer** 
 - link to /pupils/years
 - Text stayed "Learn online"
 - no external icon 
 - Legal policies reordered see [combined ticket](https://www.notion.so/oaknationalacademy/OWA-Footer-update-Hold-936d01ef23374c2b831c8a82e4f98c53)
 
 **App Header**
  - link to /pupils/years
  - no external icon
  - Aria label - `Pupils (opens in a new tab)` changed to `Pupils browse years`
  - When in section of site with `/pupils` in the url the menu is highlighted
  
 **Pupils Tab**
  - Header changed to to `Learn and revise`
  - Body changed to `We provide free online lessons with videos, quizzes, and more. Whatever the subject, from primary to GCSE, we've got you covered.`
  - Button text changed to `Find a lesson`
  - Button aria-label changed from `Learn with Oak (this will open in a new tab)` to `Find a lesson`
 

## How to test

1. Go to https://deploy-preview-2503--oak-web-application.netlify.thenational.academy
2. See the above changes and check tickets [here](https://notion.so/oaknationalacademy/OWA-Top-Global-Nav-update-34fc5c0637074b1b860c3d8c9c75ae72) and [here](https://www.notion.so/oaknationalacademy/OWA-Footer-update-Hold-936d01ef23374c2b831c8a82e4f98c53) 

## Checklist
